src/controllers/album.js

```diff
@@ -0,0 +1,105 @@
+// src/controllers/artist.js
+const db = require('../db/index.js');
+
+const createAlbum = async (req, res) => {
+  const { name, year } = req.body;
+  if (!name || !year) {
+    return res.status(400).json({ message: 'Name and year are required'});
+  }
+  const { id } = req.params;
+
+  try {
+    console.log('Before query');
```

Comment:
Console.logs are great to use when you are tracking something or when you need to debug, but they are not needed at the end result and unnecessary log messages should be deleted from the code.d